src/algebra/module/localized_module.lean

```diff
@@ -965,6 +971,215 @@ begin
   exact ⟨⟨m, s⟩, mk'_eq_iff.mpr e.symm⟩
 end
 
+@[simp]
+lemma iso_mk' (m : M) (s : S) :
+  (is_localized_module.iso S f).symm (is_localized_module.mk' f m s) = localized_module.mk m s :=
+(is_localized_module.iso S f).symm_apply_apply (localized_module.mk m s)
+
+@[simp]
+lemma lift_mk' (g : M →ₗ[R] M'')
+  (h : ∀ (x : S), is_unit ((algebra_map R (module.End R M'')) x)) (m : M) (s : S) :
+  (is_localized_module.lift S f g h) (is_localized_module.mk' f m s) = (h s).unit⁻¹ (g m) :=
+by { delta is_localized_module.lift, dsimp,
+  rw [is_localized_module.iso_mk', localized_module.lift_mk], refl }
+
 end is_localized_module
 
+section submodule
+
+variable (N : submodule R M)
+
+/-- The localization of a submodule of `M` in `localized_module S M`. -/
+def submodule.localize : submodule (localization S) (localized_module S M) :=
```

Comment:
I'm sure that I will get confused between `localize`, `localization` and `localized_...`, so I would prefer to use the same word we used before:
```suggestion
def submodule.localization : submodule (localization S) (localized_module S M) :=
```